zerver/openapi/zulip.yaml

```diff
@@ -5625,8 +5629,29 @@ paths:
           schema:
             oneOf:
               - type: string
+                enum: [newest, oldest, first_unread, date]
               - type: integer
-          example: 43
+          example: date
```

Comment:
I couldn't find any way where without changing this example value I can pass the tests in `test_openapi.py`.

Reply:
Same as https://github.com/zulip/zulip/pull/25677/commits/175286a1d5b39df042f730b8d7ace8c970537ca8#r1236887608?